lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_perform_step.jl

```diff
@@ -598,28 +598,17 @@ end
 
 ################################################################################
 
-@muladd function perform_step!(integrator, cache::Rosenbrock34ConstantCache,
-        repeat_step = false)
+@muladd function perform_step!(integrator, cache::Rosenbrock34ConstantCache, repeat_step = false)
```

Comment:
This should just be `RosenbrockConstantCache` (with renaming/combining all the caches in https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_caches.jl

Reply:
Should I combine all those caches first ?? @oscardssmith

Reply:
yeah